hic_qc.py

```diff
@@ -727,17 +727,16 @@ def pass_judgement(self):
             self.judge_bad (bool): does the hi-c library show 'bad' characteristics, e.g. zero-distance reads or too many duplicates.
             self.judge_html (str): an HTML string to put into pass/fail box
         '''
+        self.long_contacts   = float(self.stats['pairs_intracontig_hq_gt10kbp']) / self.stats['total_read_pairs_hq'] > self.min_long_contact_percentage
+        self.useful_contacts = float(self.stats['intercontig_pairs_hq'])         / self.stats['total_read_pairs_hq'] > self.min_useful_contact_percentage
+        self.same_strand_hq  = float(self.stats['pairs_on_same_strand_hq'])      / self.stats['total_read_pairs_hq'] > self.min_same_strand_percentage
```

Comment:
I think we should have a conversation here about what the denominators should be. There's a trade-off here between ease of interpretation (which favors using the same denominator for all metrics) and independence of metrics (which favors using intracontig hq count for long contacts and same strand metrics).

Reply:
Just resolving here: subsequent discussion led to a redesign, which was implemented